### ISSUE TITLE
Various Algorithms

### ISSUE THREAD

**Issue by pranjalshikhar** [Algo-Phantoms/Algo-ScriptML#3]
I can update - 

1. Linear
2. Logistic
3. KNN
4. SVM
5. Decison Tree
6. Random Forest

**Comment by rohansharma4050** [comment]
Hello I am a GSSoC'21 participant I can work on various algorithms Linear, Logistic , Naive Bayes, Random Forest ,knn, kmeans. Can you assign this to me?

**Comment by Amulya310** [comment]
Hey, I would really like to work on this issue under GSSOC'21 .Kindly consider assigning this to me!!

**Comment by muskangupta042002** [comment]
Hey, I am GSSOC'21 participant I want to work on this issue. Kindly consider assigning this to me.

**Comment by Sibasish-Padhy** [comment]
Hey I am GSSOC'21 participant i really wanna contribute to classification algorithms>Kindly consider assigning this to me

**Comment by Dishita2602** [comment]
Hello ! I can write code from scratch for Artificial Neural Networks(ANN), SVM, Random Forest, XG Boost, Ridge Regression, Decision Tree, MLP Regressor and additionally use of hyperparameter optimization and improved accuracy.

**Comment by Sibasish-Padhy** [comment]
hello!I can write code for Logistic ,KNN and linear regression from scratch and can also compare the algorithms for any dataset using balanced class accuracy(BCA)can i contribute to this project

**Comment by ashwani-rathee** [comment]
Proper Channels have been formed for this and this needs to be closed to avoid confusion
